esp/templates/customforms/index.html

```diff
@@ -31,7 +34,7 @@ <h2 id='form_title'>Form Title</h2>
 			</div>
 			
 			<div id="form_toolbox">
```

Comment:
Would it be possible to have the panel on the right move with you as you scroll down a long form? It's quite annoying to edit a form that's longer than the computer screen when you have to scroll up to get to the panel to edit and then scroll down to see the question again.